.circleci/config.yml

```diff
@@ -272,6 +283,21 @@ commands:
           rust_channel: << parameters.rust_channel >>
           platform: << parameters.platform >>
 
+  install_volta:
+    steps:
+      - run:
+          name: Install volta
+          command: |
+            curl https://get.volta.sh | bash -s -- --skip-setup
+            echo 'export VOLTA_HOME=$HOME/.volta' >> $BASH_ENV
+            echo 'export PATH=$VOLTA_HOME/bin:$PATH' >> $BASH_ENV
+      - run:
+          name: Install default versions of npm and node
+          command: |
+            volta install node@16
+            volta install npm@7
```

Comment:
We don't have to fix this at this moment, but my recommendation would be that we try to be deterministic not just always track the latest 16/7 versions — instead being explicit in order to allow for reproducible builds (e.g., even if we were to check out an old version, we wouldn't merely get the current `16` on that older build).

...and we can have Renovate actually bump these in this file.  (Perhaps using `NODE_VERSION` and `NPM_VERSION`, but maybe also just using `.npmrc` files, or the `engines` range in the `package.json`, so that it applies to local dev and CI, that might be nice too?

Thoughts, @EverlastingBugstopper?

Reply:
we pin these versions in package.json and those are updated by renovate, this just installs a default version that it will fall back to (but never actually does)